sunpy/visualization/tests/test_mapcubeanimator.py

```diff
@@ -0,0 +1,47 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import absolute_import
+import os
+
+import matplotlib.pyplot as plt
+import pytest
+
+import sunpy.map
+from sunpy.visualization import mapcubeanimator
+from sunpy.data import test
+from sunpy.tests.helpers import figure_test
+
+
+@pytest.fixture
+def map_animator():
+    testpath = test.rootdir
+    aia_file = os.path.join(testpath, "aia_171_level1.fits")
+    mapcube = sunpy.map.Map(aia_file)
+    cube = sunpy.map.MapCube(mapcube)
+    return mapcubeanimator.MapCubeAnimator(cube)
+
+
+# test for mapcubeanimator instance
+def test_mapcubeanimator_instances(map_animator):
+
+    assert isinstance(map_animator, mapcubeanimator.MapCubeAnimator)
+    assert map_animator.interval is 200
+    assert map_animator.annotate is True
+    assert map_animator.slider_ranges == [[0, 1]]
+
+    assert map_animator.timer is None
+
+    assert map_animator.active_slider == 0
+    assert map_animator.button_func == []
+    assert map_animator.remove_obj == []
+
+
+def test_updatefig(map_animator):
+    index_before = map_animator.active_slider
```

Comment:
Hey, @nabobalis Is this test okay for `updatefig`?

Reply:
Should not the index_before and index_after be different if we update the figure?

Reply:
I have incremented  the slider_index by 0 , so technically `index_before` and `index_after` should be same after we update the fig . I tried incrementing the slider_index by 1 but it gave me  an out of index error.

Reply:
Well that is because the created MapCube animator isn't a MapCube , it's just one file. I should have spotted that before.